flyteplugins/go/tasks/plugins/k8s/kfoperators/pytorch/pytorch.go

```diff
@@ -99,6 +103,25 @@ func (pytorchOperatorResourceHandler) BuildResource(ctx context.Context, taskCtx
 			return nil, flyteerr.Errorf(flyteerr.BadTaskSpecification, "Unable to create worker replica spec: [%v]", err.Error())
 		}
 
+		updateEnvVars := func(container *apiv1.Container) {
+			if container.Env == nil {
+				container.Env = make([]apiv1.EnvVar, 0, 2)
+			}
+			container.Env = append(container.Env, apiv1.EnvVar{
+				Name: pluginsK8s.FlyteInternalWorkerNameEnvVarKey,
+				ValueFrom: &apiv1.EnvVarSource{
+					FieldRef: &apiv1.ObjectFieldSelector{
+						FieldPath: "metadata.name",
+					},
+				},
+			})
+			container.Env = append(container.Env, apiv1.EnvVar{
+				Name:  pluginsK8s.FlyteInternalDistErrorStrategyEnvVarKey,
+				Value: "1",
```

Comment:
Is there any way we can use the enum value from the properties here so that everyone understands where this comes from instead of a string "1"?

Reply:
Do you mean the textual representation? As in "Earliest"?

Reply:
Ok, let me make that change.